src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ICompoundAssignmentOperation.vb

```diff
@@ -0,0 +1,348 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.Operations
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub CompoundAssignment_NullArgumentToGetConversionThrows()
+            Assert.Throws(Of ArgumentNullException)("compoundOperation", Function() TryCast(Nothing, ICompoundAssignmentOperation).GetInConversion())
```

Comment:
>TryCast [](start = 84, length = 7)

This doesn't have to be fixed, but we do not use TryCast or C# 'as' cast to perform conversions that are not expected to fail. Basically, if they are used and no null check is performed for the result, it is a red flag.